test/aggregate.test.js

```diff
@@ -738,21 +753,89 @@ describe('aggregate', () => {
     });
   });
 
-  describe('case-insensitive sorting without collation', () => {
+  describe('sorting without collation', () => {
     let collection;
     beforeAll(() => {
       collection = t.db.collection('test_aggregation_sort');
     });
 
     describe('by default...', () => {
+      const options = {
+        paginatedField: 'name',
+        sortAscending: true,
```

Comment:
Just for consistency of the spec names:
If we are _sorting without collation_ _by default..._, shouldn't this be the default value?

Reply:
I've changed the description to "without the `sortCaseInsensitive` parameter..."